WcaOnRails/app/views/competitions/my_competitions.html.erb

```diff
@@ -24,9 +25,10 @@
   <script>
     (function() {
       // Reflow table when its made visible. See:
-      //  https://github.com/cubing/worldcubeassociation.org/issues/343
+      // https://github.com/cubing/worldcubeassociation.org/issues/343
+      // changed to .trigger, see https://github.com/cubing/worldcubeassociation.org/issues/343#issuecomment-182916594
       $('#past-competitions').on('show.bs.collapse', function(e) {
-        $('#past-competitions table:not(.floatThead-table)').floatThead('reflow');
+        $('#past-competitions table:not(.floatThead-table)').trigger('reflow');
```

Comment:
Does this still work on mobile, because the plugin won't be initialized anymore, right?